### PR TITLE
Switch posts to scrollable data

### PR DESCRIPTION
**Pull request by alshakerM**
Switched posts to get data based on scrolling.

### CODE REVIEW

**Comment by alshakerM** [comment]
Fixed homePage scrolling.
Added a no more posts at the end of posts.

File: components/pages/HomePage/HomePage.js [hunk 1]
```diff
@@ -20,7 +43,13 @@ export function HomePage() {
             key={datum.post_id}
           />
         ))}
+        {itemsLeft === 0 && !isLoading && (
+          <div>
+            <h1 className={styles.noMoreText}>Sorry no more Posts</h1>
+          </div>
+        )}
```
Comment (alshakero) [inline components/pages/HomePage/HomePage.js:50]: Absolutely in love with this.

File: stores/postStore/actions.js [hunk 2]
```diff
@@ -43,21 +43,29 @@ export function setLoadingPosts(isLoading) {
...
+  return { type: 'REQUEST_POSTS', postId, pageNumber };
 }
 
-export function receivePosts(posts) {
+export function receivePosts(posts, pageNumber, itemsLeft) {
   return {
     type: 'RECEIVE_POSTS',
     posts,
+    pageNumber,
+    itemsLeft,
```
Comment (alshakero) [inline stores/postStore/actions.js:68]: Since you made `setItemsLeft` action creator, you don't need to pass `itemsLeft` here.

**Review by alshakero — APPROVED** [review]
Looks great now. Few minor comments!

File: components/pages/HomePage/HomePage.js (hunk 1 above)
Comment (alshakerM) [reply components/pages/HomePage/HomePage.js:50]: <3

**Comment by alshakerM** [comment]
Deleted some unused code.

**Pull request merged**